src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -456,7 +456,15 @@ def is_idrac(drac_client):
     requests_logger.disabled = True
 
     try:
-        drac_client.client.is_idrac_ready()
+        if drac_client.is_lifecycle_in_recovery():
+            settings = {'Lifecycle Controller State': 'Enabled'}
+            drac_client.set_lifecycle_settings(settings)
+
+            LOG.info("Waiting for lifecycle controller to get out of "
+                     "recovery mode")
+            drac_client.commit_pending_lifecycle_changes(reboot=False)
```

Comment:
Don't we still need to wait for the job to complete before returning?

Reply:
No, we don't need to wait for job completion. whenever create_config_job() method invoked and create job it will get complete in few seconds.
If you want me to add wait_for_jobs_to_complete for safer side then I need to add implementation of wait_for_jobs_to_complete() method in discover_nodes.py

Reply:
Well, I think the only reason this is working is because the next thing we do is get the model, which will wait until the iDRAC is ready.  I think I would rather explicitly wait until the job completes.

Reply:
Sure Chris! I will add wait_for_job_completion call.